src/types/profile.js

```diff
@@ -191,47 +190,10 @@ export type ProfileMeta = {|
   preprocessedProfileVersion: number,
 |};
 
-/**
- * TaskTracer data - TODO.
- */
-export type TaskTracer = {|
-  taskTable: {|
-    length: number,
-    dispatchTime: any[],
-    sourceEventId: any[],
-    sourceEventType: any[],
-    parentTaskId: any[],
-    beginTime: any[],
-    processId: any[],
-    threadIndex: any[],
-    endTime: any[],
-    ipdlMsg: any[],
-    label: any[],
-    address: any[],
-  |},
-  addressTable: {|
-    length: number,
-    address: MemoryOffset[],
-    className: IndexIntoStringTable[],
-    lib: Array<null | Lib>,
-  |},
-  threadTable: {|
-    length: number,
-    tid: any[],
-    name: any[],
-    start: any[],
-  |},
-  tasksIdToTaskIndexMap: Map<any, any>,
-  addressIndicesByLib: Map<Lib, MemoryOffset[]>,
-  stringTable: UniqueStringArray,
-  tidToThreadIndexMap: Map<any, any>,
-|};
-
 /**
  * All of the data for a processed profile.
  */
 export type Profile = {
   meta: ProfileMeta,
-  tasktracer: TaskTracer,
```

Comment:
I removed tasktracer from the processed profile definition since we no longer support it. I don't believe this change needs a profile upgrader.

Reply:
Also, I left tasktracer in the Gecko Profile definition, since that part is still valid, even if we don't use it. Although that has a very broad and simple definition.

```
export type GeckoProfile = {|
  meta: {|
    interval: Milliseconds,
    startTime: Milliseconds,
    shutdownTime: Milliseconds | null,
    abi: string,
    misc: string,
    oscpu: string,
    platform: string,
    processType: number,
    product: string,
    stackwalk: number,
    toolkit: string,
    version: number,
  |},
  libs: Lib[],
  threads: GeckoThread[],
  pausedRanges: PausedRange[],
  tasktracer?: Object,
  processes: GeckoProfile[],
|};
```

Reply:
Can you please add {| |} for the `Profile` type here while you're here?

Reply:
I filed a new bug for that as a follow-up: https://github.com/devtools-html/perf.html/issues/749